src/App.jsx

```diff
@@ -14,57 +15,72 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam'); // Початкове значення порожнє
```

Comment:
The comment suggests that the initial value should be empty, but the code sets `selectedGood` to 'Jam'. If the intention is to start with no selection, consider initializing `selectedGood` with an empty string ('').